rest-api/dao/participant_summary_dao.py

```diff
@@ -260,6 +261,7 @@ def update_enrollment_status(self, summary):
                                                          summary.physicalMeasurementsStatus,
                                                          summary.samplesToIsolateDNA)
     summary.enrollmentStatus = enrollment_status
+    summary.lastModified = clock.CLOCK.now()
```

Comment:
I don't think you want to set this here; you'll miss other times when participant summary is updated but enrollment status isn't.

Instead look for all the other places where participant summary is updated:

https://github.com/all-of-us/raw-data-repository/blob/3681280ecd45fd0f7ef419df90e76d44a337fb4b/rest-api/dao/physical_measurements_dao.py#L266
https://github.com/all-of-us/raw-data-repository/blob/3681280ecd45fd0f7ef419df90e76d44a337fb4b/rest-api/dao/biobank_order_dao.py#L182
https://github.com/all-of-us/raw-data-repository/blob/3681280ecd45fd0f7ef419df90e76d44a337fb4b/rest-api/dao/questionnaire_response_dao.py#L144
https://github.com/all-of-us/raw-data-repository/blob/3681280ecd45fd0f7ef419df90e76d44a337fb4b/rest-api/dao/participant_dao.py#L158

And make sure they all update lastModified on participant summary.

(We may need to grep around some more to make sure there aren't other places...)

Reply:
done